File: fcs59/python/peaks-arrays.py
def find_peaks(array):
    peaks = []
    for i in range(1, len(array) - 1):
        if array[i] > array[i-1] and array[i] > array[i+1]:
            peaks.append(i)
        
    return peaks

print(find_peaks([1,5,3,6,2,9]))

File: fcs59/js/peaks-arrays.js
function find_peaks(arr){
    let peaks = []
    for(let i = 1; i<arr.length; i++){
        if(arr[i] > arr[i-1] && arr[i] > arr[i+1]){
            peaks.push(i)
        }
    }

    return peaks
}

console.log(find_peaks([1,5,3,6,2,9]))